src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -180,6 +181,17 @@
         Builder realName(@Nonnull String name);
 
         /**
+         * Sets default interaction messages
+         *
+         * @param defaultInteractions A class that implements InteractionMap
```

Comment:
This `@param` feels more like the description of `InteractionMap` and perhaps could be more like:

`@param default values for interactions`
and probably a 
`@see InteractionMap`